python/ray/_raylet.pyx

```diff
@@ -480,42 +480,50 @@ cdef execute_task(
         const c_vector[CObjectID] &c_return_ids,
         c_vector[shared_ptr[CRayObject]] *returns):
 
-    worker = ray.worker.global_worker
-
-    actor_id = worker.core_worker.get_actor_id()
-    job_id = worker.core_worker.get_current_job_id()
-    task_id = worker.core_worker.get_current_task_id()
+    cdef object worker = ray.worker.global_worker
+    cdef object manager = worker.function_actor_manager
+    cdef dict execution_infos = manager.execution_infos
+    cdef CoreWorker core_worker = worker.core_worker
```

Comment:
Can you put this just under the `worker` def?

Reply:
Also please put them all under one `cdef:` block to match the rest of the functions in this file

Reply:
done